docs/pages/synth/installation_option_matrix.py

```diff
@@ -18,7 +18,8 @@
 
 
 print("If you need full-featured library, use this and read no more::", end="\n\n")
-print(f'   pip install pycyphal[{",".join(extras.keys())}]', end="\n\n")
+print(f'   pip install pycyphal[{",".join(extras.keys())}] # Linux', end="\n\n")
+print(f'   pip install pycyphal[{",".join(extras.keys())}] # macOS', end="\n\n")
```

Comment:
What is the purpose of this change? Also, it should be GNU/Linux, not Linux.

Reply:
Just showing the command is slightly different when using GNU/Linux vs macOS.

Reply:
Ah I forgot to put the right command for macOS...

Reply:
I see. The quotes are not really specific to macOS, they are required for zsh, which may be encountered in other systems as well (I am using zsh with GNU/Linux, for instance). I suggest modifying this by simply adding the quotes to the original command:

```diff
-print(f'   pip install pycyphal[{",".join(extras.keys())}]', end="\n\n")
+print(f'   pip install \'pycyphal[{",".join(extras.keys())}]\'', end="\n\n")
```